### ISSUE TITLE
Improve the usability of the rubrik_vsphere_live_mount module

### ISSUE THREAD

**Issue by Pandino** [rubrikinc/rubrik-modules-for-ansible#88]
**Is your feature request related to a problem? Please describe.**

tl;dr The `rubrik_vsphere_live_mount` module is not returning directly the name of the live mounted VM.

I am trying to setup an Ansible playbook to do some kind of automated validation of our backups (I was inspired by your PowerShell use case). Ideally, I would trigger the `rubrik_vsphere_live_mount` module to create a powered down live mount, use the `vmware_guest` module to power it up changing the network settings, do some sort of validation in the running live mount and then unmount it. 

However with the current Rubrik modules, I find this workflow quite cumbersome and please correct me if there is an easier way to do it.

To start, the `rubrik_vsphere_live_mount` simply return the CDM API response. So I need to call the `rubrik_job_status` module to know when the live mount is ready. This last module is not returning any result that is easily usable with the other Ansible modules.

At this point my only way to know the name of the new machine is to call the `rubrik_get_vsphere_live_mount_names`, but then I have to choose a machine from a list and there is always the remote chance that the latest VM in the list is not the one created by our script.

**Describe the solution you'd like**

I think that an Ansible module shouldn't be a simple facade for the API. So I would expect the `rubrik_vsphere_live_mount` module to return the name of the newly created VM like this:

```yaml
- name: Live mount the latest snapshot
  rubrik_vsphere_live_mount:
    vm_name: "{{ vm_name }}"
    power_on: False
    wait_for_completion: True
  register: live_mount
  delegate_to: localhost

 - name: Setup and start the new VM
   vmware_guest:
     name: "{{ live_mount.instance.name }}"     
     networks:
       - name: "{{ test_network }}"
         ip: "{{ test_ip }}"
         netmask: 255.255.255.0
     status: poweredon
     wait_for_ip_address: True
   delegate_to: localhost

- name: Unmount the live mount
  rubrik_vsphere_live_unmount:
    mounted_vm_name: "{{ live_mount.instance.name }}"
    force: True
  delegate_to: localhost
  
```

In this example, the `wait_for_completion` argument will wait for the job to finish and it will make the module return the results from the endpoint `v1/vmware/vm/{live_mount_VM_id}` under the key `instance`. Without the `wait_for_completion` argument the `instance` key could be simply empty.

**Describe alternatives you've considered**

Another possible solution is a new module that always waits for the job completion and returns only the `v1/vmware/vm/{live_mount_VM_id}` endpoint data.
